### DISCUSSION TITLE
The update of do not update

### DISCUSSION THREAD

**Discussion by hermitpopcorn** [Docile-Alligator/Infinity-For-Reddit#1266] [Q&A] [answered]
![IMG_20221205_100610](https://user-images.githubusercontent.com/16042129/205541023-c9989dfd-2e9d-459a-9c7c-bf2046ae097d.jpg)

What's going on?

**Comment by IverCoder** [comment, marked as answer]
The developer accidentally pushed the beta release `5.4.0-beta1` to production and had to make another update `5.3.5` to prevent the beta build from being released too early. `5.3.5` and `5.3.4` are practically the same, the only difference is the version code. So there's no reason to update.
